### PR TITLE
feat(react-component-typing-avatar): add hover tooltip of name

### PR DESCRIPTION
**Pull request by adamweeks**

### CODE REVIEW

**Comment by adamweeks** [comment]
Deploy preview ready!

Built with commit bef3ad68a9aed677420540ac0b8d3739c3e27622

https://deploy-preview-578--hopeful-hamilton-4090b9.netlify.com

File: packages/node_modules/@ciscospark/react-component-typing-avatar/src/styles.css [hunk 1]
```diff
@@ -21,3 +21,43 @@
...
+  left: -56px;
+  width: 130px;
+  height: 24px;
+  line-height: 24px;
+  background-color: #000;
+  border-radius: 5px;
+  opacity: 0;
+}
+
+.tooltip::after {
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/react-component-typing-avatar/src/styles.css]: totally stole this from https://leaverou.github.io/bubbly/

File: packages/node_modules/@ciscospark/react-component-typing-avatar/src/index.js [hunk 2]
```diff
@@ -16,6 +16,11 @@ function TypingAvatar(props) {
   }
   return (
     <div className={classNames(`ciscospark-typing-avatar`, styles.typingAvatar)}>
+      <div className={classNames(`ciscospark-typing-avatar-tooltip`, styles.tooltip)}>
+        <div className={classNames(`ciscospark-typing-avatar-tooltip-text`, styles.tooltipText)}>
+          {name}
+        </div>
+      </div>
```
Comment (bzang) [inline packages/node_modules/@ciscospark/react-component-typing-avatar/src/index.js]: Should this be it's own component so we can use it in other places?

Comment (adamweeks) [reply packages/node_modules/@ciscospark/react-component-typing-avatar/src/index.js]: I'm going to be implementing it in the group icon as well and try to figure out how to make it its own component. Next PR :)

**Pull request merged**